software/rfcs/rocketCtl.md

```diff
@@ -0,0 +1,68 @@
+# RocketCtl
+
+RocketCtl or RocketControl is our system for configuring Apex systems before launch. It can control much of the setup of the avionics without needing us to recompile the code. 
```

Comment:
note - ithis is mportant as UKRA consider any changes to code a different system, so if we want to get our own avionics approved for use then we need to have a fixed codebase, so any alterations will need to be done in another way